salt/_macros/kubectl.jinja

```diff
@@ -91,10 +91,10 @@
 wait-for-{{ deployment }}-deployment:
```

Comment:
Not for this PR, but at some moment we should refactor this and create a `wait_for_deployment` state in Python.